### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Leon-Mbegera**
In this project I :
 - Created a new repo
 - Added a html file
 - Added a css file
 - Added a test file
 - And a linter file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header class="one">
```
Comment (Abdona) [inline index.html]: Use h1 tag instead of header tag.because h1 is more descriptive .

**Review by Abdona — CHANGES_REQUESTED** [review]
Please accept my review.
Make the changes and then resubmit your code review.

**Comment by Leon-Mbegera** [comment]
Hi
I believe I have addressed the changes regarding the descriptive tags.

**Comment by Leon-Mbegera** [comment]
All the changes are successful.
Now I merge the pull request.

**Pull request merged**